flutter/lib/src/sentry_flutter.dart

```diff
@@ -115,9 +123,17 @@ mixin SentryFlutter {
     // in errors.
     integrations.add(LoadReleaseIntegration(packageLoader));
 
+    integrations.add(MobileVitalsIntegration(nativeWrapper));
```

Comment:
This works not only on Android/iOS/macOS but also on web, Windows and Linux?

Reply:
This is only available for the native iOS/Android. Will fix this.